### PR TITLE
Modify release-ts-api gha

### PR DESCRIPTION
**Pull request by 0xverin**
### Context

After migrating client-api to npm registry and using built worker image directly for deployment, basic modifications need to be made to this gha.

### CODE REVIEW

File: .github/workflows/release-ts-api-package.yml [hunk 1]
```diff
@@ -18,25 +25,20 @@ jobs:
...
+          docker pull litentry/litentry-parachain:$DOCKER_TAG
 
       - name: Re-tag docker image
         run: |
-          docker tag litentry/litentry-worker:$WORKER_DOCKER_TAG litentry/litentry-worker:latest
-          docker tag litentry/litentry-cli:$WORKER_DOCKER_TAG litentry/litentry-cli:latest
-          docker tag litentry/litentry-parachain:$PARACHAIN_DOCKER_TAG litentry/litentry-parachain:latest
+          docker tag litentry/litentry-worker:$DOCKER_TAG litentry/litentry-worker:latest
+          docker tag litentry/litentry-cli:$DOCKER_TAG litentry/litentry-cli:latest
+          docker tag litentry/litentry-parachain:$DOCKER_TAG litentry/litentry-parachain:latest
```
Comment (Traf333) [inline .github/workflows/release-ts-api-package.yml]: these changes might be adjusted once we[ resolve polling ](https://litentry.slack.com/archives/C05328L0GAF/p1716829164458979)

**Review by Traf333 — CHANGES_REQUESTED** [review]
I'm not sure that even after merging [the PR](https://github.com/litentry/litentry-parachain/pull/2759), your changes would work. From what I understood, we don't have `litentry/identity-cli`, but you can extract it from `litentry/identity-worker` 

@Kailai-Wang can you correct me if I'm wrong?

**Comment by 0xverin** [comment]
> I'm not sure that even after merging [the PR](https://github.com/litentry/litentry-parachain/pull/2759), your changes would work. From what I understood, we don't have `litentry/identity-cli`, but you can extract it from `litentry/identity-worker`
> 
> @Kailai-Wang can you correct me if I'm wrong?

I saw `identity-cli` on Docker Hub when I was making modifications.
https://hub.docker.com/r/litentry/identity-cli

**Comment by Traf333** [comment]
Oh, my bad. It was addressed within this PR https://github.com/litentry/litentry-parachain/pull/2761

**Review by Traf333 — APPROVED** [review]
🚀

**Pull request merged**